integration-authorization/pom.xml

```diff
@@ -25,6 +25,12 @@
         </dependency>
 
         <dependency>
+            <groupId>com.google.http-client</groupId>
+            <artifactId>google-http-client-jackson</artifactId>
+            <version>1.15.0-rc</version>
+        </dependency>
```

Comment:
Copy this dependency to <dependencyManagement> section on the parent pom and update the version to 1.22.0

Example:
 ```
<dependency>
    <groupId>com.google.http-client</groupId>
    <artifactId>google-http-client-jackson2</artifactId>
    <version>1.22.0</version>
</dependency>
```

In this pom file, you should change the artifactId to 'google-http-client-jackson2' and remove the version.